### ISSUE TITLE
UI Update made saved graphql request with multiple queries unusable

### ISSUE THREAD

**Issue by gyles** [getinsomnia/insomnia#5631]
### Expected Behavior

When I open an existing graphql request with multiple queries (and/or mutations) inside, the new UI does not let me easily switch to different queries.

### Actual Behavior

The previous UI allows you to click on any query/mutation inside the text area and it will automatically select that query to be executed.

### Reproduction Steps

1. Open an existing GraphQL request with multiple queries inside 
    e.g., 
    ```
    query personById(id: 'person-id') {
        ...
    }
    query personByName(name: 'person-name') {
        ...
    }
    ```
2. User is not able to switch between queries because the dropdown is disabled. The user needs to modify the request to enable the dropdown.
3. User is not able switch between queries by clicking on any part of the query. (as in the old UI)

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.7.0

### What operating system are you using?

macOS

### Operating System Version

macOS Ventura v13.0.1

### Installation method

Download from insomnia.rest

### Last Known Working Insomnia version

before the UI change

**Comment by filfreire** [comment]
Hi @gyles thanks for reporting this. This particular issue should be fixed in the [latest patch release 2022.7.1](https://github.com/Kong/insomnia/releases/tag/core%402022.7.1)

**Comment by gyles** [comment]
@filfreire Is there a way to switch between queries within the same request? There was no need to use a dropdown before.
